webapp/public/javascripts/controllers.js

```diff
@@ -28,7 +29,7 @@ lunchrControllers.controller('MainPageController', ['$scope', '$http', '$state',
 
             if(/^[_a-z0-9]+(\.[_a-z0-9]+)*@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})$/.test($scope.email) == false)
             {
-                $scope.errorMessages = "Invalid Email";
+                $scope.errorMessages = "Incorrect email";
                 return;
             }
```

Comment:
Numbers and special characters should be allowed. The requirements are no uppercase letters and an @ symbol. If a period is included there must be following text.